src/pivot-table/components/__tests__/DimensionCell.test.tsx

```diff
@@ -32,7 +32,23 @@ describe('DimensionCell', () => {
     };
 
     dataModel = {
-      pivotData: { matrix: [], topMatrix: [], leftMatrix: [], nbrTopRows: 0, nbrLeftColumns: 0 },
+      pivotData: {
```

Comment:
Would be nice to have this mocked if it is used in multiple places, not relevant for this pr though

Reply:
Agree and I've thought about it, but for now it's now that widely used so it will have to wait.